### PR TITLE
Added role examples

### PR DESCRIPTION
**Pull request by ATGE**
issue #239

### CODE REVIEW

File: content/python/role_examples.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+title: "Role Examples"
+description: "Examples of how Roles works"
+date: "2019-04-09"
+classes: 
+    - "SoftLayer_User_Permission_Role"
+    - "SoftLayer_User_Permission_Group"
+    - "SoftLayer_User_Permission_Action"
+tags:
+    - "user"
+    - "users"
```
Comment (allmightyspiff) [inline content/python/role_examples.md]: just use the `user` tag

File: content/python/role_examples.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    "name": "name role",
+    "description": "test user permission role"
+}
+
+client = SoftLayer.create_client_from_env()
+
+try:
+
+    createRole = client['User_Permission_Role'].createObject(templateObject)
+    print(createRole)
```
Comment (allmightyspiff) [inline content/python/role_examples.md]: should this be pprint ?

File: content/python/role_examples.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+            'HARDWARE_VIEW'
+        ]
+        return actions
+
+
+if __name__ == "__main__":
+    main = Group()
+    # The user permission group id on which will be added permissions.
+    group_id = 12345678
+    main.add_permission(group_id)
```
[the remark this answers is not in the document]
Comment (allmightyspiff) [reply content/python/role_examples.md]: removing `actions_keynames` and changing the main function to this I think would be ideal:

```python
if __name__ == "__main__":
    main = Group()
    actions = [
            'TICKET_VIEW',
            'TICKET_EDIT',
            'TICKET_ADD',
            'HARDWARE_VIEW'
    ]
    # The user permission group id on which will be added permissions.
    group_id = 12345678
    main.add_permission(group_id, actions)
```

File: content/python/role_examples.md (hunk 2 above)
Comment (ATGE) [reply content/python/role_examples.md]: Yes, is right

File: content/python/role_examples.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ATGE) [reply content/python/role_examples.md]: You are right, I will do it that way

**Pull request merged**